src/AppKit/Enums.cs

```diff
@@ -877,6 +886,7 @@ public enum NSWindowStyle : int {
 		DocModal	       					= 1 << 6,
 		NonactivatingPanel     				= 1 << 7,
 		TexturedBackground     				= 1 << 8,
+		[Deprecated (PlatformName.MacOSX, 10, 14)]
```

Comment:
any suggestions for a replacement in the header / docs ?

Reply:
The header said it's just deprecated and no effect.